doc/rose-rug-task-run.html

```diff
@@ -317,6 +317,11 @@ <h3 id="rose-task-run.command">Command Selection</h3>
               task-run</code>, the arguments are run as a shell
               command.</li>
 
+              <li>If the <code>ROSE_APP_COMMAND_KEY</code> environment variable
+              is set, the command specified in the <var>[command]KEY</var>
+              setting in the application configuration whose KEY matches it
+              is used.</li>
```

Comment:
Should the environment variable come third to allow the CLI to selectively overwrite a general setting?